src/components/InfoTooltip/Tooltip.style.tsx

```diff
@@ -84,9 +84,20 @@ export const StyledMarkdown = styled(MarkdownBody)`
   }
 `;
 
-export const StyledSpan = styled.span`
+const tooltipAnchorText = css`
   text-decoration: underline;
+  text-decoration-style: dashed;
+  text-underline-offset: 3px;
   cursor: pointer;
-  color: ${COLOR_MAP.GRAY_BODY_COPY};
+  color: ${COLOR_MAP.GREY_4};
+`;
+
+export const SmallTooltipAnchorText = styled.span`
+  ${tooltipAnchorText};
   font-size: 0.875rem;
 `;
+
+export const LargeTooltipAnchorText = styled.span`
+  ${tooltipAnchorText};
+  font-size: 1rem;
```

Comment:
Hrm...  It looks like the only thing `SmallTooltipAnchorText` and `LargeTooltipAnchorText` override is the font-size...   Can we just rely on the component using the tooltip to set the correct font size?  E.g. if I want a big font I could do:

```
<span style={{fontSize: '30px'}}><TextToolTip ... /></span>
```

In fact, I think maybe that already happens?  If I just remove the `font-size` setting here and on line 97 so that `SmallTooltipAnchorText` and `LargeTooltipAnchorText` are identical and don't set the font size, I think everything still renders exactly the same?  So maybe we don't need these and the `altAnchorComponent` thing?

Reply:
ahh yes, if i dont pass a font size, it grabs it from the parent. updating!